__tests__/end-to-end.js

```diff
@@ -589,7 +607,7 @@ describe('end-to-end', () => {
       } else log.warn('OpenTripPlanner not accepting requests. Start it up for deployment tests!!')
     }
     log.info('Launching chromium for testing...')
-    browser = await puppeteer.launch(puppeteerOptions)
+    browser = await launch(puppeteerOptions)
```

Comment:
I'm getting an error here when I run this locally:
```
 ● end-to-end › admin › should allow admin user to create another user

    TypeError: Cannot read property '_launcher' of undefined

      608 |     }
      609 |     log.info('Launching chromium for testing...')
    > 610 |     browser = await launch(puppeteerOptions)
          |                                     ^
      611 |     page = await browser.newPage()
```

Reply:
Try running `rm -rf node_modules && yarn && yarn run test-end-to-end`.

Reply:
Nope. Are you sure you don't have any local changes? It works when I import puppeteer instead of `import {launch}`

Reply:
Just made a little refactor. Try again.